src/main/features/claimant-response/views/confirmation/proposed-alternate-payment-plan-to-business-defendant.njk

```diff
@@ -0,0 +1,28 @@
+<h2 class="heading-medium">Send financial details</h2>
+<p>{{ t('You need to send the defendant’s financial details to the court.') }}</p>
+
+<p>{{ t('Make sure you include your claim number: {{ claimNumber }}.', { claimNumber: claim.claimNumber }) }}</p>
+
+<p>{{ t('The court will review the details to see if the defendant can afford your repayment plan.') }}</p>
+
+<h2 class="heading-medium">{{ t('Use this address') }}</h2>
+
+<p>{{ t('The defendant should have sent you the details - send them to:') }}</p>
+
+<p>
```

Comment:
use a div with the `keep-formatting panel` classes as can be found in the send-your-response-by-email.njk